Makefile

```diff
@@ -23,11 +23,11 @@ REAL_CLONE_URL = $(if $(CLONE_URL),$(CLONE_URL),$(DEFAULT_CLONE_URL))
 # Run code quality checks
 style_check:
 	black --check .
-	isort --check .
```

Comment:
Why do we remove isort? Is ruff ruff a replacement for isort? But why do we still leave isort in the setup?

Reply:
Yes, ruff also manages imports the same way as isort does but it does it much faster.
You're right, I just removed isort from the dependencies!